web/packages/extension/assets/css/popup.css

```diff
@@ -22,6 +22,10 @@ button {
     color: black;
 }
 
+#version-text {
+    text-align: center;
```

Comment:
Hello and welcome - this looks great! A little thing I noticed here is that your `version-text` element is a `span`, not a block-level element like a `div`, so your `text-align: center` styling has no effect:
![image](https://user-images.githubusercontent.com/71368227/206939808-4a5077ef-0510-40ad-94fc-780940b16c51.png)
If you change it to a div, it looks like what you probably intended:
![image](https://user-images.githubusercontent.com/71368227/206939904-a716ebc5-fe95-4abf-9ecd-9e68c2672055.png)

Reply:
Looks good to me! 👍